resource_activity/models/resource_activity.py

```diff
@@ -157,8 +157,8 @@ def _compute_push2sale_order(self):
             if activity.sale_orders:
                 registrations_need_push = any(
                     activity
-                    .registrations
-                    .mapped('need_push'))
+                        .registrations

```

Comment:
did black say this?

Reply:
nope, it's the "Reformat File" from PyCharm :sweat: `black` will fix this

Reply:
here comes `black` !